docket/plugins/config_scripts.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+import asyncio
+from difflib import get_close_matches
+from typing import cast
+
+import asyncpg
+import crescent
+import hikari
+
+from docket.config import CONFIG
+from docket.database.models.guild import Guild
+from docket.database.models.script import Script
+from docket.errors import DocketBaseError, ScriptNotFound
+from docket.plugins._checks import has_guild_perms
+from docket.plugins._group import docket_group
+
+plugin = crescent.Plugin(
+    "config-scripts", command_hooks=[has_guild_perms(hikari.Permissions.MANAGE_GUILD)]
+)
+scripts = docket_group.sub_group("scripts", "Manage scripts")
+
+
+async def wait_for_script(ctx: crescent.Context) -> str | None:
+    def predicate(message: hikari.GuildMessageCreateEvent) -> bool:
+        return (
+            message.channel_id == ctx.channel_id
+            and message.author.id == ctx.user.id
+            and (message.content is not None or message.message.attachments is not None)
+        )
+
+    try:
+        msg = await ctx.app.wait_for(
+            hikari.GuildMessageCreateEvent, timeout=30, predicate=predicate
+        )
+    except asyncio.TimeoutError:
+        return None
```

Comment:
```suggestion
    try:
        msg = await ctx.app.wait_for(
            hikari.GuildMessageCreateEvent, timeout=30, predicate=predicate
        )
    except asyncio.TimeoutError:
        await ctx.edit("Script creation timed out.")
        return
```

Reply:
Doing this will make the commands themselves seem a little weird. It won't be clear that the original response is being edited. I think it's best left as-is.

Reply:
Alright that makes sense